sklearn/tests/test_docstring_parameters.py

```diff
@@ -43,6 +44,7 @@
 ]
 
 
+@fails_if_pypy  # this segfaults with PyPy-6.0
```

Comment:
This test it not relevant for scikit-learn users (it checks docstring formatting), so I think we can safely skip it in PyPy.